Framework/PythonInterface/plugins/algorithms/VelocityAutoCorrelations.py

```diff
@@ -308,7 +308,7 @@ def auto_correlation(self,u):
 
     def fold_correlation(self,w):
         # Folds an array with symmetrical values into half by averaging values around the centre
-        right_half=w[len(w)/2:]
+        right_half=w[len(w)//2:]
         left_half=w[:int(np.ceil(len(w)/2.0))][::-1]
```

Comment:
That's correct but it's not related to what I'm fixing here. _i.e._ I don't care.